middlewares/inviteTokenValidation.js

```diff
@@ -0,0 +1,40 @@
+const jwt = require("jsonwebtoken");
+const Map = require("../models/Map");
+
+const { ERROR_MESSAGE } = require("../constants");
+
+const { INVITATION_SECRET_KEY } = process.env;
+
+const validateInviteToken = async (req, res, next) => {
+  const { id } = req.params;
+
+  try {
+    const currentMap = await Map.findById(id).exec();
+
+    const verifyToken = token => {
+      try {
+        return jwt.verify(token, INVITATION_SECRET_KEY).hasOwnProperty("email");
```

Comment:
```suggestion
       jwt.verify(token, INVITATION_SECRET_KEY;
       return true;
```

hasOwnProperty 메소드를 한 번 타는 것보다 토큰을 verify 했을 때 문제가 없다면 true를 리턴해주는건 어떨까요..?
inviteToken에는 페이로드로 무조건 이메일이 들어가기 때문에 hasOwnProperty 메소드를 한 번 더 타주는 건 의미가 없는게 아닐까 싶어서요..!